mne/utils/docs.py

```diff
@@ -3548,14 +3522,45 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
 
 docdict['time_bandwitdh_tfr'] = """
 time_bandwidth : float
-    Time x (Full) Bandwidth product. Should be ``≥ 2.0``.
-    Choose this along with ``n_cycles`` to set the desired frequency
-    resolution. The number of good tapers (least leakage from far away
-    frequencies) is chosen automatically based on this to
-    floor ``time_bandwidth - 1``.
-    For example, with ``freq = 20 Hz`` and ``n_cycles = 10``, we get
-    ``time = 0.5 s``. If ``time_bandwidth = 4.``, then frequency smoothing
-    is ``(4 / time) = 8 Hz``."""
+    Time x (Full) Bandwidth product. Should be ``≥ 2.0``. The frequency
+    resolution achieved depends on ``n_cycles`` and ``time_bandwdith``. For a
+    given time-window length, a finer frequency resolution is achieved by
+    decreasing ``time_bandwidth`` at the cost of the number of good tapers. See
+    notes for additional information."""
+
+docdict['time_bandwitdh_tfr_note'] = """
+The frequency resolution achieved depends on 2 arguments:
+
+- ``n_cycles``, the number of cycles which defines the time-window length.
+- ``time_bandwitdh``, the product of the time-window length (in seconds) and of
+  the frequency resolution (in Hz), the bandwidth. This product can be seen as
+  the surface of the window on the time/frequency plane.
+
+The frequency resolution is
+:math:`fq_{resolution} = {time-bandwitdh} / {time-window}`. Once ``n_cycles``
+is set, the time-window length is fixed. Increasing ``time_bandwidth`` will
+increase the bandwidth, and vice-versa.
+
+For example, with a fixed length time-window of ``0.5`` seconds defined by
+``freqs=np.arange(1, 6, 2)`` and ``n_cycles=freqs / 2``:
+
+- if ``time_bandwith = 4``, the frequency resolution is :math:`4 / 0.5 = 8 Hz`
+- if ``time_bandwith = 2``, the frequency resolution is :math:`2 / 0.5 = 4 Hz`
+
+.. note::
+
+    Frequency resolution, or bandwitdh, is also refered to as frequency
+    smoothing. The bandwitdth is centered around the frequencies of interest
+    ``freqs`` and will smooth together the frequencies in its range.
+    For example, a bandwidth of 4 Hz at a frequency of interest of 10 Hz will
+    smooth together the frequencies ± 2 Hz around 10 Hz, i.e. between 8 Hz and
+    12 Hz.
+
+However, the gained frequency resolution reduces the number of good tapers,
```

Comment:
in multitapering you often don't want to increase the frequency resolution but increase smoothing (independent of window length), so I doub't someone would use it if their primary goal is to increase frequency resolution.

Reply:
also, the description is very long now and better fits an example \ short tutorial, I think. I'd keep the argument description short now and add a short intro tutorial on multitapering in a next PR.

Reply:
oh, that's a note, then the length is likely ok.